spring-cloud-gcp-logging/src/test/java/org/springframework/cloud/gcp/logging/TraceIdLoggingWebMvcInterceptorTests.java

```diff
@@ -33,10 +33,14 @@
 
 	public static final String TEST_TRACE_ID = "105445aa7843bc8bf206b120001000";
 
+	public static final String TEST_TRACE_ID_2 = "205445aa7843bc8bf206b120001000";
```

Comment:
We should probably call this `ZIPKIN_TRACE_ID_HEADER_VALUE` and the one below `GCP_TRACE_ID_HEADER_VALUE` for clarity.